config/locales/en.yml

```diff
@@ -1212,6 +1217,7 @@ en:
   notice_to_many_principals_to_display: "There are too many results.\nNarrow down the search by typing in the name of the new member (or group)."
   notice_unable_delete_time_entry: "Unable to delete time log entry."
   notice_unable_delete_version: "Unable to delete version."
+  notice_value_information: "Your %{key_name} is: %{value}"
```

Comment:
I find this rather hard to translate. Given this is my only context, I have no idea where translation will be used. This is especially due to the poor naming of the key (`value_information`).

How about nesting it as proposed above? (e.g. `my_account.access_token.show_key_message`)